spk/salt-master/Makefile

```diff
@@ -1,38 +1,54 @@
 SPK_NAME = salt-master
-SPK_VERS = 2015.5.11
-SPK_REV = 9
+SPK_VERS = 3003.3
+SPK_REV  = 1
 SPK_ICON = src/salt-master.png
 
-DEPENDS =
+SPK_DEPENDS = "python38"
+
+DEPENDS = cross/patch
+
+BUILD_DEPENDS  = cross/python38 cross/setuptools cross/pip
+BUILD_DEPENDS += cross/wheel
+BUILD_DEPENDS += cross/salt
+
+include ../../mk/spksrc.archs.mk
+
+# Requirements file generation
+# /usr/local/python3/bin/python3 -mvirtualenv --python=python3  salt-env
+# virtualenv --python=python3  salt-env
+# source salt-env/bin/activate
+# pip install salt
+# pip freeze > requirements.txt
 WHEELS = src/requirements.txt
-SPK_DEPENDS = "python>2.7.8-10"
 
-MAINTAINER = SynoCommunity
-DESCRIPTION = Salt, a new approach to infrastructure management, is easy enough to get running in minutes, scalable enough to manage tens of thousands of servers, and fast enough to communicate with those servers in seconds.
-RELOAD_UI = yes
+MAINTAINER   = SynoCommunity
+DESCRIPTION  = Salt, a new approach to infrastructure management, is easy enough to get running in minutes, scalable enough to manage tens of thousands of servers, and fast enough to communicate with those servers in seconds.
+RELOAD_UI    = yes
```

Comment:
This is an old setting that doesn't do anything anymore
```suggestion

```

Reply:
fixed